ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+<!--
+Thank you for reporting an issue! For feature requests, scroll down.
+
+###########################
+Regarding problem reports:
+###########################
+
+Before you submit a problem report, please do the following:
+
+  * Search the list of existing issues to see if someone has reported the issue already.
+  * Update to the latest master branch of SeqAn and see if the issue persists.
+
+If you already know how to fix the issue, you can directly create a pull-request instead of an issue. See the guide on [CONTRIBUTING](./CONTRIBUTING.md) for more details.
+-->
+
+## Platform
+
+  - SeqAn version: <!-- enter version here -->
+  - Operating system: <!-- enter output of `uname -a` here-->
+  - Compiler: <!-- enter output of `/path/to/compiler --version` here -->
+
+## Description
+
+### How to repeat the problem
+
+<!-- step to repeat the problem, code snippets... -->
+
+### Expected behaviour
+
+<!-- what you thought would happen -->
+
+### Actual behaviour
+
+<!-- what happens -->
+
+
+<!--
+###########################
+Regarding feature requests:
+###########################
+
+For feature requests you can remove all of the above, but clearly indicate that this is a feature request in the name, e.g. "[feature-request] add support for Fast5 format".
+
+One of the following will happen to feature requests:
+
+  * We will "accept" it and someone will start working on it for the next sprint [the issue will be kept open].
+  * We will convert it to a card in our [long-term project planning](https://github.com/seqan/seqan3/projects), but close the issue as no-one can work on it immediately.
+  * The feature request is invalid or not in the scope of the next releases and will be closed.
```

Comment:
* We will "accept" it and will include it for the current or the next release.
  * We will "accept" it but freeze the feature as there are no resources available to work on it. You are also very welcome to make a PR for it. Please read the [CONTRIBUTING](./CONTRIBUTING.md) guidlines for more details.
  * The feature request is invalid and will be closed.

Reply:
I have a slight different workflow in mind. Maybe we should discuss this the next strategy meeting?

Reply:
ok